### PR TITLE
Added primary_metastore_latency

### PR DESCRIPTION
**Pull request by patduin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ For more information please refer to the main [Apiary](https://github.com/Expedi
 | bastion_ssh_key_secret_name | Secret name in AWS Secrets Manager which stores the private key used to log in to bastions. The secret's key should be `private_key` and the value should be stored as a base64 encoded string. Max character limit for a secret's value is 4096. | string | `` | no |
 | cpu | The number of CPU units to reserve for the Waggle Dance container. Valid values can be 256, 512, 1024, 2048 and 4096. Reference: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html | string | `1024` | no |
 | default_latency | Latency used for the primary metastore, and other other metastores that don't override it in their own configurations. See `latency` parameter in https://github.com/ExpediaGroup/waggle-dance/blob/main/README.md. | number | `0` | no |
+| primary_metastore_latency | Same as default_latency | number | `0` | no |
```
Comment (barnharts4) [inline README.md:14]: This description isn't exactly true.  The two descriptions need to be updated to explain what primary is used for, and what default_latency is used for (only used if either primary or other metastores don't explicitly set latency).

**Review by barnharts4 — CHANGES_REQUESTED** [review]

File: templates.tf [hunk 2]
```diff
@@ -130,7 +130,7 @@ data "template_file" "federation_yaml" {
   vars = {
     primary_metastore_host             = var.primary_metastore_host
     primary_metastore_port             = var.primary_metastore_port
-    primary_metastore_latency          = var.default_latency
+    primary_metastore_latency          = var.primary_metastore_latency == "" ? var.default_latency : var.primary_metastore_latency
```
[the remark this answers is not in the document]
Comment (patduin) [reply templates.tf:133]: doh

File: README.md (hunk 1 above)
Comment (patduin) [reply README.md]: reworded

File: variables.tf [hunk 3]
```diff
@@ -250,3 +250,9 @@ variable "default_latency" {
   default     = 0
   description = "HMS latency (in ms.) that Waggledance will tolerate.  See \"latency\" parameter in https://github.com/ExpediaGroup/waggle-dance/blob/main/README.md."
 }
+
+variable "primary_metastore_latency" {
+  type        = number
+  default     = 0
+  description = "HMS latency (in ms.) that Waggledance will tolerate.  See \"latency\" parameter in https://github.com/ExpediaGroup/waggle-dance/blob/main/README.md."
```
[the remark this answers is not in the document]
Comment (patduin) [reply variables.tf]: I agree I changed it and the other one, I could rename `default_latency` to `default_federate_metastore_latency` if that makes sense it's technically backward incompatible but perhaps we can get away with it???

[the remark this answers is not in the document]
Comment (barnharts4) [reply variables.tf]: I think it's good the way it is now.

**Review by barnharts4 — APPROVED** [review]

**Review by abhimanyugupta07 — APPROVED** [review]

**Pull request merged**